src/antsibull/changelog.py

```diff
@@ -489,6 +498,133 @@ def __init__(
         self.collection_metadata = collection_metadata
 
 
+def _get_removal_entry(
+    collection: str,
+    removal: RemovalInformation,
+) -> tuple[ChangelogFragment, str] | None:
+    if removal.announce_version is None:
+        return None
+
+    sentences = []
+    link = ""
+    if removal.discussion:
+        link = f" (`{removal.discussion} <{removal.discussion}>`__)"
+
+    if removal.reason == "deprecated":
+        sentences.append(f"The ``{collection}`` collection has been deprecated.")
+        sentences.append(
+            f"It will be removed from Ansible {removal.major_version} if no one"
+            " starts maintaining it again before Ansible {removal.major_version}."
+        )
+        sentences.append(
+            "See `the removal process for details on how this works"
+            " <https://docs.ansible.com/ansible/devel/community/collection_contributors/"
+            "collection_package_removal.html#canceling-removal-of-an-unmaintained-collection"
+            f">`__{link}."
+        )
+
+    if removal.reason == "considered-unmaintained":
+        sentences.append(
+            f"The ``{collection}`` collection is considered unmaintained"
+            f" and will be removed from Ansible {removal.major_version}"
+            f" if no one starts maintaining it again before Ansible {removal.major_version}."
+        )
+        sentences.append(
+            "See `the removal process for details on how this works"
+            " <https://docs.ansible.com/ansible/devel/community/collection_contributors/"
+            "collection_package_removal.html#canceling-removal-of-an-unmaintained-collection"
+            f">`__{link}."
+        )
+
+    if removal.reason == "renamed":
+        sentences.append(
+            f"The collection ``{collection}`` has been renamed to ``{removal.new_name}``."
```

Comment:
This can be simple past tense.

```suggestion
            f"The collection ``{collection}`` was renamed to ``{removal.new_name}``."
```